cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaMirrorMaker2AssemblyOperatorMockTest.java

```diff
@@ -130,7 +130,7 @@ public void afterEach() {
 
         LOGGER.info("Reconciling initially -> create");
         Promise created = Promise.promise();
-        Checkpoint async = context.checkpoint();
```

Comment:
Another potential risk in a helper method that didn't help measure tests passing or not.